### PR TITLE
V1.9.0 rel

### PR DESCRIPTION
**Pull request by d3zd3z**
Preparations for releasing v1.9.0-rc1.

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -3,6 +3,32 @@
 - Table of Contents
 {:toc}
 
+## Version 1.9.0
+
+The 1.9.0 release of MCUboot contains various bug fixes, improves
+support on some recent targets, and adds support for devices with a
+write alignment larger than 8.
```
Comment (utzig) [inline docs/release-notes.md:10]: But `imgtool` allows only up to 32, not sure larger than this was tested...

**Review by utzig — APPROVED** [review]
LGTM

File: docs/release-notes.md (hunk 1 above)
Comment (nvlsianpu) [reply docs/release-notes.md:10]: up to 32 B: https://github.com/mcu-tools/mcuboot/blob/main/boot/bootutil/include/bootutil/bootutil_public.h#L86

File: docs/release-notes.md [hunk 2]
```diff
@@ -3,6 +3,32 @@
...
+
+This change introduces a potentially incompatible change to the format
```
Comment (nvlsianpu) [inline docs/release-notes.md:12]: I would mention that new feature is accessible over `MCUBOOT_BOOT_MAX_ALIGN`, if this is undefined or defined to 8 the mcuboot is backward compatible.

File: docs/release-notes.md (hunk 1 above)
Comment (utzig) [reply docs/release-notes.md:10]: Thanks, I didn't remember that, we should cite this in the release notes.

**Pull request merged**